lean4-infoview/src/infoviewApi.ts

```diff
@@ -40,10 +46,10 @@ export interface EditorApi {
    */
   //emulateServerNotification(method: string, params: any): Promise<void>;
 
-  /** Apply edits to the workspace. */
-  applyEdits(edits: TextDocumentEdit[]): Promise<void>
+  /** Insert text into a document. When `pos` is not present, write at the current cursor location. */
+  insertText(text: string, kind: TextInsertKind, pos?: TextDocumentPositionParams): Promise<void>
```

Comment:
Sadly the range of things which can be done with `TextEdit` without knowing the file contents (and for now we don't really need to track them in the infoview) is quite small, so reinstated this API.